docs/install/install.rst

```diff
@@ -24,8 +26,9 @@ Prerequisites
    .. tab-item:: NVIDIA
       :sync: nvidia
 
-      Check the system requirements in the
-      `NVIDIA CUDA Installation Guide <https://docs.nvidia.com/cuda/cuda-installation-guide-linux/>`_.
+      On NVIDIA GPUs HIP requires unified memory. All CUDA enabled NVIDIA
```

Comment:
```suggestion
      With NVIDIA GPUs, HIP requires unified memory. All CUDA-supported NVIDIA
```

Reply:
Fixed the first part. NVIDIA itself talks of "CUDA-enabled" GPUs, if you follow the link, so I thought it would be better to use that term.